src/Layer/Manager/AbstractManager.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Layer\Manager;
+
+/**
+ * Class AbstractManager
+ * @package Layer\Manager
+ */
+abstract class AbstractManager implements ManagerInterface
```

Comment:
In the fig-standards repository, is the file with naming conventions for code released by PHP-FIG (https://github.com/php-fig/fig-standards/blob/master/bylaws/002-psr-naming-conventions.md). Do we need know and use them?